docs/source/installation/micromamba-installation.rst

```diff
@@ -200,13 +200,28 @@ Build from source
 To build from source, install the development dependencies, using a Conda compatible installer
 (``conda``/``mamba``/``micromamba``/``rattler``/``pixi``).
 
+
+
+First create a virtual environment with necessary dependencies using  ``micromamba`` like follows:
+
 .. code-block:: bash
 
   micromamba create -n mamba --file dev/environment-micromamba-static.yml
-  micromamba activate -n mamba
+
+Then activate the created a virtual environment with necessary dependencies using  ``micromamba`` like follows:
+
+.. code:: bash
+
+    micromamba activate -n mamba
+
+
+.. warning::
+
+   It is supposed to build mamba package from a freshly downloaded source code, building from source that has been previously built with a different cmake configuration will lead to inconsistent result for the current cmake configuration
 
 Use CMake from this environment to drive the build:
 
+First configure cmake as follows:
```

Comment:
```suggestion
First configure and generate the build system:
```

Reply:
I wonder if it would better call this step as "Set the cmake configuation" as Build configurations allow a project to be built in different ways for debug, optimized, or any other special set of flags. (ref. https://cmake.org/cmake/help/book/mastering-cmake/chapter/Getting%20Started.html)

Reply:
IMHO it does more that setting so what about something in the line of "Generate the CMake build system for the given build configuration" ?

Reply:
```suggestion
Generate the CMake build system for the given build configuration:
```

Reply:
```suggestion
First generate the CMake build system for the given build configuration:
```